utils/getInvoices/tests/getInvoices.discount.test.ts

```diff
@@ -6,135 +6,452 @@ import { ServiceType } from '@utils/constants'
+        price: 1000,
+        sum: 1000,
+      })
     })
+    it('should load when service = { rentPrice: 0 }, company = {discount: 0 }', () => {
+      const service: Partial<IService> = {
+        rentPrice: 0,
+      }
+      const company: Partial<IRealestate> = {
+        discount: 0,
+      }
 
-    expect(invoices).not.toContainEqual(
-      expect.objectContaining({ type: ServiceType.Discount })
-    )
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).toContainEqual({
+        type: ServiceType.Discount,
+        price: 0,
+        sum: 0,
+      })
+    })
+    it('should NOT load when service = { rentPrice: 1000 }, company = null ', () => {
+      const service: Partial<IService> = {
+        rentPrice: 1000,
+      }
+      const company: Partial<IRealestate> = null
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Discount })
+      )
+    })
+    it('should load when service = { ... }, company = { discount: 1000 } ', () => {
+      const service: Partial<IService> = {
+        rentPrice: 1000,
+      }
+      const company: Partial<IRealestate> = {
+        discount: 1000,
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).toContainEqual({
+        type: ServiceType.Discount,
+        price: 1000,
+        sum: 1000,
+      })
+    })
+    it('should load when service = { rentPrice: 0 }, company = { discount: 1000 } ', () => {
+      const service: Partial<IService> = {
+        rentPrice: 0,
+      }
+      const company: Partial<IRealestate> = {
+        discount: 1000,
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).toContainEqual({
+        type: ServiceType.Discount,
+        price: 1000,
+        sum: 1000,
+      })
+    })
+    it('should load when service = { rentPrice: null }, company = { discount: 1000 } ', () => {
+      const service: Partial<IService> = {
+        rentPrice: null,
+      }
+      const company: Partial<IRealestate> = {
+        discount: 1000,
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).toContainEqual({
+        type: ServiceType.Discount,
+        price: 1000,
+        sum: 1000,
+      })
+    })
+    it('should load when service = { rentPrice: undefined }, company = { discount: 1000 } ', () => {
+      const service: Partial<IService> = {
+        rentPrice: undefined,
+      }
+      const company: Partial<IRealestate> = {
+        discount: 1000,
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).toContainEqual({
+        type: ServiceType.Discount,
+        price: 1000,
+        sum: 1000,
+      })
+    })
```

Comment:
Remove this tests, they also have no puprose and can be handled in `should load when service = { ... }, company = { discount: 1000 }`